docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+mkdocs==1.5.0
+mkdocs-material==8.2.6
+mkdocs-spellcheck==0.2.1
+markdown_include==0.6.0
+pygments==2.15.0
+jinja2==3.0.3
+markdown==3.3.7
```

Comment:
`mkdocs-material` already [depends on](https://github.com/squidfunk/mkdocs-material/blob/43ed6fac8dfc5385a9bdf22d8bf207c868c23a80/requirements.txt#L22) `mkdocs`, `pygments`, `jinja2`, and `markdown`, so we don't need to explicitly specify them in our own `requirements.txt` (and `mkdocs` itself also [depends on](https://github.com/mkdocs/mkdocs/blob/7186f4ce9c88e7c445ef90fee85c065ca297a7c8/pyproject.toml#L35) `jinja2` and `markdown`)

```suggestion
```